native/packages/react-native-bpk-component-input/src/BpkInput.js

```diff
@@ -0,0 +1,146 @@
+/*
+ * Backpack - Skyscanner's Design System
+ *
+ * Copyright 2017 Skyscanner Ltd
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import {
+  View,
+  TextInput,
+Image,
+  Platform,
+  StyleSheet,
+} from 'react-native';
+import React from 'react';
+import PropTypes from 'prop-types';
+
+const IOS_TOKENS = require('bpk-tokens/tokens/ios/base.react.native.common.js');
+const ANDROID_TOKENS = require('bpk-tokens/tokens/android/base.react.native.common.js');
+
+const tokens = Platform.select({
+  ios: () => IOS_TOKENS,
+  android: () => ANDROID_TOKENS,
+})();
+
+const styles = StyleSheet.create({
+  input: {
+    borderColor: tokens.colorGray100,
+    borderWidth: 1,
+    borderRadius: tokens.spacingSm, // TODO SWAP OUT FOR CORRECT TOKEN
+    padding: tokens.spacingSm * 3,
+    flexDirection: 'row',
+  },
+  text: {
+    fontSize: tokens.textLgFontSize,
```

Comment:
I reckon that whenever you are using `textLgFontSize` or equivalent tokens directly you should be using the Text component.

Reply:
Also if we don't end up using `BpkText` we should at least have a dedicated token i.e `INPUT_TEXT_FONT_SIZE`